tests/unit/services/label_data_parsing/test_label_utils.py

```diff
@@ -261,19 +260,6 @@ def test_integration_of_label_parsing_in_kili_labels_assert_types(mocker):
     )
 
 
-def test_integration_of_label_parsing_in_kili_asset_assert_types(mocker):
```

Comment:
where have those tests been moved to?

Reply:
So the test about the return types have been integrated to in `tests/integration/entrypoints/client/queries/test_user_facing_queries.py::test_assets_query_return_type` because there were already a test on the asset query return type based on its input arguments.

and the test about the parsing when calling the function has been moved to `tests/integration/use_cases/test_asset.py` because it is not a unit but an integration test. We don't test the parsing class but the fact that when calling the use case, it will go though the parsing process

Reply:
tests/integration/entrypoints/client/queries/test_user_facing_queries.py is not here

Reply:
-> `tests/integration/entrypoints/client/queries/test_assets_queries.py`
i don't know where I found test_user_facing_queries